architecture/additional_concepts/authorization.adoc

```diff
@@ -525,6 +525,25 @@ privileged         true      [*]       RunAsAny    RunAsAny           RunAsAny
 restricted         false     []        MustRunAs   MustRunAsRange     MustRunAs   RunAsAny    <none>     false            [configMap downwardAPI emptyDir persistentVolumeClaim secret]
 ----
 
+ifdef::openshift-origin,openshift-enterprise[]
+[IMPORTANT]
+====
+Do not modify the default SCCs. Customizing the default SCCs can lead to issues
+when {product-title} is upgraded. Instead,
+xref:../../admin_guide/manage_scc.adoc#creating-new-security-context-constraints[create
+new SCCs].
+====
+endif::openshift-enterprise,openshift-origin[]
```

Comment:
@bfallonf - brice, wrong endif (or wrong ifdef :P). Can you please fix it today?

Reply:
@vikram-redhat fixed in #9519